index.html

```diff
@@ -284,7 +284,7 @@ <h4 class="heading"><i class="fas fa-mobile"></i>YOUR APPS WOULD BE PUSHED TO PL
                   </div> 
   
                   <div class="paragraph-wrapper">
-                      <p class="paragraph">Hmmm, it seems an another cool reason.. isn’t it? We will push your apps on <strong>Google PlayStore</strong>  	and in the future, maybe <strong>AppStore</strong> as well :) So that you can only focus on developing the app 	and leaving all monetary and maintenance burdens to us<strong>(WE DO LOVE BURDENS).</strong></p>
+                      <p class="paragraph">We will push your apps on <strong>Google PlayStore</strong>  	and in the future, maybe <strong>AppStore</strong> as well :) So that you can only focus on developing the app 	and leaving all monetary and maintenance burdens to us</p>
```

Comment:
Hmmm, it seems another cool reason.. isn’t it?

We can keep this line like this.